src/pages/applications.js

```diff
@@ -315,6 +316,17 @@ const emergingProjects = [
       {label: 'Website', url: 'https://merbridge.io/'},
     ],
   },
+  {
+    logoUrl: 'https://github.com/deepflowys/deepflow',
+    name: 'DeepFlow',
+    logo: deepflowLogo,
+    title: 'Highly Automated Observability Platform powered by eBPF',
+    description: `DeepFlow is a highly automated observability platform built for cloud native developers. Based on eBPF, DeepFlow innovatively implements an automated distributed tracing mechanism: AutoTracing. Microservice processes, service mesh sidecars, and network interfaces along the way are included as tracing spans, for every distributed transaction, without any code instrumentation. Besides, with the AutoMetrics mechanism implemented on cBPF and eBPF, DeepFlow can automatically generate golden RED metrics for any process in cloud native environment.`,
```

Comment:
Small nit:

`Besides, with the AutoMetrics mechanism implemented on cBPF and eBPF, DeepFlow can automatically generate golden RED metrics for any process in cloud native environment.`

Maybe change into:

`DeepFlow can automatically generate golden RED metrics for any process in cloud native environment.`

It's a bit unclear what to make out of the cBPF in the text there (I presume you mean old-style socket filters..).

Please also rebase once again w/o a merge commit. So it's all a single clean commit on top of latest master.

Reply:
@borkmann thanks for your suggestions.

all done